OsmAnd/res/values/strings.xml

```diff
@@ -106,12 +106,12 @@
     <string name="map_center_widget">Coordinates: Map center</string>
     <string name="current_location_widget">Coordinates: Current location</string>
     <string name="external_sensors_plugin_name">External Sensors</string>
-    <string name="external_sensors_plugin_description">Monitoring of heart rate, bicycle speed, bicycle power, and cadence. Requires your device to be connected to respective sensors via the ANT+ wireless personal network protocol.</string>
+    <string name="external_sensors_plugin_description">Monitoring of heart rate, bicycle speed, power, and cadence via ANT+ connection to respective sensors.</string>
```

Comment:
But the connection needs to be configured on the device level, not in OsmAnd, hence the original seems clearer.

Reply:
That was lost on me. Added "in Android".